bundles/binding/org.openhab.binding.ddwrt/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: openHAB ddwrt Binding
+Bundle-SymbolicName: org.openhab.binding.ddwrt
+Bundle-Version: 1.9.0.qualifier
+Bundle-Activator: org.openhab.binding.ddwrt.internal.ddwrtActivator
```

Comment:
Please make sure all of your class and interface names start with a capital letter.